procedures/unit-testing-tracing.ipf

```diff
@@ -75,6 +75,56 @@ static Function/S GetTaggedFunctionName(string procWin)
 	return FUNCTION_TAG_PREFIX + Hash(procWin, 1) + "_IGNORE"
 End
 
+static Function InitFuncLocations(variable numProcs)
+	DFREF dfr = GetPackageFolder()
+	WAVE/WAVE wv = GetFuncLocations()
+
+	Redimension/N=(numProcs, -1) wv
+
+	Make/FREE/T/N=0 emptyText
+	Make/FREE/N=0 emptyVar
+
+	wv[][%FUNCLIST] = emptyText
+	wv[][%FUNCSTART] = emptyVar
+End
+
+static Function/WAVE GetFuncLocations()
+	string name = "FuncLocations"
+	DFREF dfr = GetPackageFolder()
+	WAVE/WAVE/Z wv = dfr:$name
+
+	if(WaveExists(wv))
+		return wv
+	endif
+
+	Make/O/WAVE/N=(0, 2) dfr:$name/WAVE=wv
+	SetDimLabel UTF_COLUMN, 0, FUNCLIST, wv
+	SetDimLabel UTF_COLUMN, 1, FUNCSTART, wv
+
+	return wv
+End
+
+static Function InitProcSizes(variable numProcs)
+	DFREF dfr = GetPackageFolder()
+	WAVE wv = GetProcSizes()
+
+	Redimension/N=(numProcs) wv
+End
+
+static Function/WAVE GetProcSizes()
+	string name = "ProcSizes"
+	DFREF dfr = GetPackageFolder()
+	WAVE/Z wv = dfr:$name
+
+	if(WaveExists(wv))
+		return wv
+	endif
+
+	Make/O/N=(0) dfr:$name/WAVE=wv
```

Comment:
why /O here?
You can also omit the parenthesis for `/N=` is the dimension is a literal single number.